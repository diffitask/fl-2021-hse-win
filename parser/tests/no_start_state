 Alphabet: 2 -- ("c", "d")
 Q: 2
 start:
 T: 1 -- 1
 edges: (0; 1; "c", "d")