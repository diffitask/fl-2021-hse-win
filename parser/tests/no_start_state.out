Error: Syntax error
Error: Start state of the automat has not been detected
